### PR TITLE
デザインをCSSに分離 (#3)

### PR DESCRIPTION
**Pull request by gomama**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,9 +2,13 @@
...
     <meta charset="utf-8">
-    <title>〄</title>
+    <link rel="stylesheet" type="text/css" href="styles/main.css">
+    <title>〄Senbei Holdings〄</title>
 </head>
-<body style="background-color: #415661;">
-    <marquee style="color: azure;">〄</marquee>
+<body>
+    <div class="marquee">
+        <p class="jis">Welcome to Senbei Holodings!</p>
```
Comment (highemerly) [inline index.html:10]: Typo.
s/Holodings/Holdings.

File: styles/main.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    color: azure;
+    font-size: 10em;
+}
+
+.marquee {
+    overflow:hidden;
+    margin:1em 0;
+  }
+  
+  .marquee p {
```
Comment (highemerly) [inline styles/main.css]: Fix indent.
L15-L24.

File: styles/main.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    background-color: #415661;
+}
+
+.jis {
+    color: azure;
+    font-size: 10em;
+}
+
+.marquee {
+    overflow:hidden;
```
Comment (highemerly) [inline styles/main.css]: Insert space.

Before:
```
overflow:hidden
```

After:
```
overflow: hidden
```

Check other places from the same point.
Thank you.

File: styles/main.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    margin:1em 0;
+  }
+  
+  .marquee p {
+    margin:0;
+    padding-left:100%;
```
Comment (highemerly) [inline styles/main.css]: Why padding-left is needed?

File: index.html [hunk 5]
```diff
@@ -2,9 +2,13 @@
 <html>
 <head>
     <meta charset="utf-8">
-    <title>〄</title>
+    <link rel="stylesheet" type="text/css" href="styles/main.css">
```
Comment (highemerly) [inline index.html]: Could you change the folder name to "assets"? I think style sheet will be only one file, so that "styles" folder is not suitable.

**Review by highemerly — COMMENTED** [review]
Some comments.

**Comment by highemerly** [comment]
#5 も同時にFixしようとしていますね

File: index.html (hunk 1 above)
Comment (gomama) [reply index.html:10]: 拝承

File: styles/main.css [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    display:inline-block;
+    white-space:nowrap;
+    animation-name:marquee;
+    animation-duration:10s;
+      animation-timing-function:linear;
+      animation-iteration-count:infinite;
+  }
+  
+  @keyframes marquee {
+    from { transform:translate(0);     }
```
Comment (highemerly) [inline styles/main.css]: FIx indent.

```
from {
   trans~~~
}
to {
   trans~~~
}
```

File: styles/main.css (hunk 2 above)
Comment (gomama) [reply styles/main.css]: 拝承

File: styles/main.css (hunk 3 above)
Comment (gomama) [reply styles/main.css]: 拝承

File: styles/main.css (hunk 6 above)
Comment (gomama) [reply styles/main.css]: 拝承

File: index.html (hunk 5 above)
Comment (gomama) [reply index.html]: 拝承

File: styles/main.css (hunk 4 above)
Comment (gomama) [reply styles/main.css]: 不明

**Comment by highemerly** [comment]
Thanks a lot @gomama !!

**Pull request merged**